vendor/github.com/ONSdigital/go-ns/handlers/accessToken/handler.go

```diff
@@ -0,0 +1,44 @@
+package accessToken
+
+import (
+	"context"
+	"errors"
+	"net/http"
+
+	"github.com/ONSdigital/go-ns/common"
+	"github.com/ONSdigital/go-ns/log"
+)
+
+// CheckHeader is a wrapper which adds a accessToken from the request header to context if one does not yet exist
+func CheckHeader(h http.Handler) http.Handler {
```

Comment:
Is `CheckHeader` an accurate description of what this handler does? `ForwardAccessTokenIfExists` or something along those lines feels more appropriate (to me anyway).

Reply:
Just see this is a vendor file. Can ignore my comment. But we should change it.

Reply:
@daiLlew , just saw this, this was me (as is the next one - poor Dan). These are in the accessToken go-ns package so it'd be `accessToken.CheckHeader` and `accessToken.CheckCookie`, it's descriptive in that context but happy to change if you think it needs to be clearer.